org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/junit/JUnitFragment.xtend

```diff
@@ -18,40 +18,51 @@ import org.eclipse.xtext.xtext.generator.XtextGeneratorNaming
 import org.eclipse.xtext.xtext.generator.model.FileAccessFactory
 import org.eclipse.xtext.xtext.generator.model.JavaFileAccess
 import org.eclipse.xtext.xtext.generator.model.TypeReference
+import org.eclipse.xtext.xtext.generator.util.BooleanGeneratorOption
 
 import static extension org.eclipse.xtext.GrammarUtil.*
 
 /**
  * @since 2.14
  */
 class JUnitFragment extends AbstractStubGeneratingFragment {
-	
+
 	@Inject extension XtextGeneratorNaming
 	@Inject FileAccessFactory fileAccessFactory
-	
+
 	@Accessors(PUBLIC_SETTER)
 	boolean useDeprecatedClasses
-	
+
 	@Accessors(PUBLIC_SETTER)
 	boolean skipXbaseTestingPackage
-	
+
 	JUnitVersion junitVersion = JUnitVersion.JUNIT_4
 	
-	def void setJunitVersion (String version) {
-		junitVersion = JUnitVersion.fromString(version)
+	final BooleanGeneratorOption generateXtendStub = new BooleanGeneratorOption(true);
```

Comment:
This field here smells like a hack.
Can we instead call `setGenerateXtendStub(false)` from our constructor and use the BooleanGenOption from the super class?

Reply:
something we should try. this idea did not came to my mind.

Reply:
It didn't occur to me either.